system/security/LdapSecurity/ldapconnection.cpp

```diff
@@ -761,6 +763,11 @@ class CLdapConfig : implements ILdapConfig, public CInterface
         return m_group_basedn.str();
     }
 
+    virtual bool getDisableCreateAdminGroup()
```

Comment:
In general it is clearer to not include negatives in function names and member variables.  E.g.

virtual bool getAllowCreateAdminGroup(), otherwise your test conditions (as below) become double negatives.

Reply:
Converted to enabled semantics